scannermaven.properties

```
@@ -6,30 +6,35 @@ organizationUrl=https://www.sonarsource.com
 homepageUrl=https://docs.sonarqube.org/latest/analysis/scan/sonarscanner-for-maven/
 issueTrackerUrl=https://sonarsource.atlassian.net/jira/software/c/projects/MSONAR/issues
 sourcesUrl=https://github.com/SonarSource/sonar-scanner-maven
-archivedVersions=3.6.1,3.7,3.8,3.9
-publicVersions=3.9.1
+archivedVersions=3.6.1,3.7,3.8,3.9,3.9.1
+publicVersions=3.10
+
+3.10.description=Support Maven 4
+3.10.date=2023-09-15
+3.10.changelogUrl=https://sonarsource.atlassian.net/projects/MSONAR/versions/12662/tab/release-report-all-issues
```

This page is not publicly available and requires login:
```suggestion
3.10.changelogUrl= https://sonarsource.atlassian.net/issues/?jql=fixVersion%20%3D%2012662
```